CHANGELOG.md

```diff
@@ -1298,6 +1304,7 @@ Please read the changelog and test in your environment.
  - The initial versions  were rapildy iterated to deliver
    a working ingress controller.
 
+[2.0.0]: https://github.com/kong/kubernetes-ingress-controller/compare/2.0.0-beta.2...2.0.0
```

Comment:
```suggestion
[2.0.0]: https://github.com/kong/kubernetes-ingress-controller/compare/2.0.0-beta.2...2.0.0-ga
```

Reply:
OK. Within this table, 2.0.0 literally indicates a tag, please keep 2.0.0 as it is. I do not mean to have a tag name 2.0.0-ga since that is not a convention.

Reply:
A bunch of them _were_ wrong though, since we started using `v` prefixes, so 3daeda74